include/net/nrf_cloud.h

```diff
@@ -712,6 +712,20 @@ int nrf_cloud_uninit(void);
  */
 int nrf_cloud_print_details(void);
 
+/**
+ * @brief Print details about cloud connection after connected.
+ *
+ * Some information is only available after the device is connected.
+ * When using MQTT the stage and the tenant will be printed.
+ * Call this function after your cloud event handler receives the first
+ * NRF_CLOUD_EVT_RX_DATA_SHADOW event.
```

Comment:
i don't recall, does a `NRF_CLOUD_EVT_RX_DATA_SHADOW` event always occur?  if there is no config section for example.
`NRF_CLOUD_EVT_READY` should be fine?

Reply:
I had earlier tried after `NRF_CLOUD_EVT_READY` but the device had not yet received the shadow (and thus the tenant) so it printed an error.

Yes, `NRF_CLOUD_EVT_RX_DATA_SHADOW` works whether or not a config section is present -- confirmed on a device without a config section.

Reply:
looking at the code, the tenant id should be obtained before the `NRF_CLOUD_EVT_READY` event.
it is obtained/set along with the topic prefix, which causes the FSM to advance to the `STATE_UA_PIN_COMPLETE` state.

Reply:
Thanks -- turns out I had tried after cloud_connected, not cloud_ready.